concepts/kwdef/about.md

```diff
@@ -0,0 +1,131 @@
+# About
+
+The macro `Base.@kwdef` can be used to automatically define constructors that take keyword arguments for the definition of a [struct][struct].
+It also allows one to set default values for fields.
```

Comment:
```suggestion
These keyword argument constructors can also set default values for fields.
```

Otherwise I'd think that the positional constructors have defaults too.